.github/workflows/main.yml

```diff
@@ -0,0 +1,171 @@
+        Write-Host "::set-output name=semver::v$($env:GitVersion_SemVer)"                
+        $fileName = "dummy-v$($env:GitVersion_SemVer)"
+        echo "SemVerMMP: v$($env:GitVersion_MajorMinorPatch)"
+        Write-Host "Filename: '$fileName'" 
+
+
+    - name: Setup msbuild
+      uses: microsoft/setup-msbuild@v1.1
+      with:
+          vs-version: '12.0'
+     
+      
+    - name: Restore Packages
+      run: Nuget restore SourceGrid.sln
+
+    - name: Build
+      run: msbuild -m -t:Rebuild /p:Configuration=Release /p:Platform="Any CPU" /fl SourceGrid.sln
+        
+    
+    - name: Archive Build Log
+      uses: actions/upload-artifact@v2
+      if: ${{ success() || failure() }}
+      with:
+        name: Compile_Solution_log
+        path: |
+          *.log    
+    
+      
+      
+    - name: Create zip
+      id: packageBuildResults
+      run: |
+        $sourceFolder = Join-Path $env:GITHUB_WORKSPACE "out" | Join-Path -ChildPath "net*" 
+        $outFolder = Join-Path $env:GITHUB_WORKSPACE "out" | Join-Path -ChildPath "sourcegrid"
+        New-Item -ItemType Directory -Force -Path $outFolder
+        $fileName = "sourcegrid-2.0.0.zip"
+        Write-Host "Filename: '$fileName'"
+        Write-Host "sourceFolder: '$sourceFolder'"
+        Write-Host "Outfolder: '$outFolder'"
+        Write-Host "::set-output name=sourcegrid::$($fileName)"
+        Remove-Item $sourceFolder\**\*.pdb -Force -Recurse
+        $outPath = Join-Path $outFolder $fileName
+        Compress-Archive -DestinationPath $outPath -Path $sourceFolder -CompressionLevel Optimal
+      
+        
+    - name: Archive Build Output
+      uses: actions/upload-artifact@v2
+      with:
+         name: sourcegrid
+         path: |
+            out/sourcegrid
+
+
+    - name: Create Nuget Packages 
+      id: createNupkg
+      run: |
+        nuget pack .nuget\SourceGrid.nuspec -Version 2.0.0
+        Write-Host "::set-output name=nupkg-sourcegrid::sourcegrid.2.0.0.nupkg"
+        
+    - name: Archive NuGet Packages
+      uses: actions/upload-artifact@v2
+      with:
+        name: nuget-sourceGrid
+        path: |
+          *.nupkg
+
+  release:
+    if: github.ref == 'refs/heads/Github_runner'    
+    runs-on: windows-latest
+    needs: build
+  
+    steps:
+      - name: Download Build Output
+        uses: actions/download-artifact@v2
+        with:
+          name: sourcegrid
+
+      - name: Download NuGet Package
+        uses: actions/download-artifact@v2
+        with:
+          name: nuget-sourceGrid
+
+      - name: Debug
+        run: |
+          tree
+      
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
+        with:
+          tag_name: v2.0.0
+          #tag_name: ${{ needs.build.outputs.semver }}
+          release_name: Release v2.0.0
+          body: |
+            ${{ github.event.head_commit.message }}
```

Comment:
what commit message will be shown here?

Reply:
It will show the last commit message

Reply:
e.g-Merge pull request #105 from siemens/development